routes/users.js

```diff
@@ -5,6 +5,45 @@ const usersController = require('../controllers/usersController')
 const userValidator = require('../middlewares/validators/user')
 
 router.post('/', authenticate, userValidator.createUser, usersController.addNewUser)
+
+/**
+ * @swagger
+ * /users/self:
+ *   patch:
+ *     summary: Use to update the user data.
```

Comment:
Please add `requestBody` after this line.
Refer here https://swagger.io/docs/specification/basic-structure/

Reply:
Done with requested changes. Thank you!